app/graphql/schema/type-definitions.js

```diff
@@ -150,22 +132,46 @@ type Preference {
   content: String
 }
 
-type CustomerQuery {
+type Status {
+  code: Int
+  success: Boolean
+  message: String
+}
+
+type Responses {
+  internalUser: Boolean
+  name: String
+  heading: String
+  body: String
```

Comment:
I think we need a timestamp for each response. So when a response is created we should record the time.

Reply:
Ahh got it, I'm trying to do something where I don't use the Cosmos timestamp, so will include that as well.

Reply:
So it seems like I'm able to get the `statusCode` when performing mutations but not for queries? I've chosen to remove the status type definition from the queries altogether as like you said I'd prefer to avoid hardcoding as well.

Reply:
Yep, so I think using the timestamp that Cosmos automatically generates (`_ts`) isn't the best option to work with in the end. I've found that when I update a ticket with a new response, the timestamp in Cosmos updates too because it recognises that an existing item was modified. I tried specifying to use the "original" timestamp, but it will update in Cosmos regardless every time a new response is upserted. 

Instead what I've done is delete the `convertCosmosTimestamp` util and have replaced it with a `generateTimestamp` function. This will just generate a timestamp in a readable format e.g. `dd/mm/yyyy, hh:mm:ss` and the timestamp will persist and remain the same for the ticket and its responses. This way the timestamp also appears more readable via the emulator also.

Reply:
Nice work 👏🏻